config/initializers/i18n_custom_interpolation.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# This custom interpolation is used to format interpolated dates in error messages
+# so they align to the NHS design system.
+module I18n
+  module CustomInterpolation
```

Comment:
Since we're including it below, can we customise the name as it's specifically about reformatting the date? Or do you expect this to be used for other types of values too?

Reply:
Ya, fair. I've renamed it to `NHSDesignSystemDateInterpolation`.